src/main/java/com/casper/sdk/model/clvalue/cltype/CLTypeMap.java

```diff
@@ -29,6 +29,25 @@ public void setKeyValueTypes(CLTypeMapEntryType keyValueTypes) {
 
     private CLTypeMapEntryType keyValueTypes;
 
+    @Override
+    public boolean isDeserializable() {
+
+        if (getKeyValueTypes().keyType instanceof CLTypeAny || getKeyValueTypes().valueType instanceof CLTypeAny) {
+            // The map contains an 'Any' type therefore cannot be deserialized
+            return false;
+        } else if (getKeyValueTypes().valueType instanceof AbstractCLTypeWithChildren) {
+           return getChildTypes().stream().allMatch(childType -> {
```

Comment:
extract into another method or class

Reply:
The list and map have slightly different behaviour. Personally I find DRY for two similar methods is overkill. If there were 3 instance of this I'd consider it